aws/resource_aws_transfer_server.go

```diff
@@ -30,6 +30,30 @@ func resourceAwsTransferServer() *schema.Resource {
 				Computed: true,
 			},
 
+			"certificate": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
```

Comment:
The [`UpdateServer`](https://docs.aws.amazon.com/transfer/latest/userguide/API_UpdateServer.html) API supports a `Certificate` field, is `ForceNew: true` required (i.e. can the certificate be updated without error)?